packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -151,9 +165,13 @@ class PerformanceOverlayLayer extends Layer {
   /// The rectangle in this layer's coodinate system that the overlay should occupy.
   Rect overlayRect;
 
-  /// A mask specifying the statistics to display.
+  /// The mask is created by shifting 1 by the index of the specific
+  /// [PerformanceOverlayOption] to enable.
   final int optionsMask;
 
+  /// The rasterizer threshold is an integer specifying the number of frame
+  /// intervals that the rasterizer must miss before it decides that the frame
+  /// is suitable for capturing an SkPicture trace for further analysis.
```

Comment:
I'm not sure this feature makes sense anymore.  We should replace it with something that integrates with the diagnostic server.